spec/features/support/changing_a_users_associated_organisations_spec.rb

```diff
@@ -183,7 +183,7 @@ def and_i_dont_see_a_button_to_associate_the_school
 
   def and_i_select_a_new_responsible_body_name
     select other_local_authority.name, from: 'support-user-responsible-body-form-responsible-body-field'
-    associated_organisations_page.submit_responsible_body_name.click
+    click_on 'Move'
```

Comment:
could we not use SitePrism object?

Reply:
This spec undergoes quite a lot of change on #1000 so I'd prefer to correct this on that PR.